augly/image/functional.py

```diff
@@ -64,6 +64,7 @@ def apply_lambda(
         except AttributeError:
             func_kwargs["aug_function"] = type(aug_function).__name__
     func_kwargs = imutils.get_func_kwargs(metadata, func_kwargs)
+    src_mode = image.mode
```

Comment:
```suggestion
            func_kwargs["aug_function"] = type(aug_function).__name__

    func_kwargs = imutils.get_func_kwargs(metadata, func_kwargs)
    src_mode = image.mode
```
(not caused by your diff but i think this would be more readable)